lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -203,7 +213,7 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
             this.loadProperties(node);
         });
 
-        if (this.displayPredictions) {
+        if (this.isContentEnrichmentFlagOn && this.displayPredictions) {
```

Comment:
Maybe a common function, to return if contentEnrichment is enabled or not? So something like 
```typescript
private function enableContentEnrichment(): boolean {
    return this.isContentEnrichmentFlagOn && this.displayPredictions;
{
```
That way, any future changes in enabling/disabling of the feature can also be controlled from a common place

Reply:
the feature flag should be removed when we decide to publish the feature, so only `this.displayPredictions` will remain

Reply:
Oh, in that case its fine. We don't need another function